test/integration/recipe.test.js

```diff
@@ -57,6 +57,34 @@ describe('Recipes', function () {
       expect(responseRest).to.deep.equal(newRecipe)
     })
 
+    it('should get recipe by id - 200', async function () {
+      const newRecipe = {
+        externalId: 'foobar3000',
+        name: 'foobar3000',
+        imageAttachmentId: '00000000-0000-1000-8000-000000000000',
+        material: 'foobar3000',
+        alloy: 'foobar3000',
+        price: 'foobar3000',
+        requiredCerts: [{ description: 'foobar3000' }],
+        supplier: 'valid-1',
+      }
+
+      const recipe = await postRecipeRoute(newRecipe, app, authToken)
+      const response = await getRecipeByIdRoute(app, recipe.body.id, authToken)
+      expect(response.status).to.equal(200)
+    })
+
+    it('should fail to get by id - 404', async function () {
+      const response = await getRecipeByIdRoute(app, '11111111-ba46-4871-9d91-63248be7b884', authToken)
+      expect(response.status).to.equal(404)
+    })
+
+    // it.only('should fail when invalid id is given - 404', async function () {
+    //   const response = await getRecipeByIdRoute(app, '', authToken)
+    //   //console.log(response)
+    //   expect(response.status).to.equal(404)
+    // })
```

Comment:
Test with invalid Id should produce a `400` error (which you shouldn't need extra code for as it's handled by the OpenAPI validation). Good test to have